sound/soc/sof/Makefile

```diff
@@ -10,7 +10,7 @@ snd-sof-objs +=	ipc3.o ipc3-loader.o ipc3-topology.o ipc3-control.o ipc3-pcm.o\
 endif
 ifneq ($(CONFIG_SND_SOC_SOF_INTEL_IPC4),)
 snd-sof-objs += ipc4.o ipc4-loader.o ipc4-topology.o ipc4-control.o ipc4-pcm.o\
-		ipc4-mtrace.o
+		ipc4-mtrace.o ipc4-telemetry.o
```

Comment:
I would have separate patch for the core and for the Intel implementation, but if others are OK, then commit title should be updated:
`ASoC: SOF: ipc4/Intel: ...`

Reply:
The commit message does not make much sense?

But the order and split somehow got confusing, I would suggest:
- patch 1-3 is OK
- patch 4: should be this patch + the introduction of ipc4-telemetry.h as
           ASoC: SOF: ipc4: Add definition of telemetry slot for exception handling
           Explain what the structs are and the core only provides the exception debugfs file, platforms will do the panic decoding.
- patch 5: enable the debugfs (the last patch from the series)
- patch 6: The Intel implementation of the exception decoding
- patch 7-8: mtl/tgl to use the decoding
- patch 9: handle the EXCEPTION_CAUGHT

Does this makes sense?

Reply:
Thanks!